src/components/organisms/UiList/UiList.vue

```diff
@@ -103,17 +103,22 @@ const itemsToRender = computed(() => (props.items.map((item, key) => {
     name: item.name || `list-item-${key}`,
   };
 })));
+/* eslint-disable @typescript-eslint/no-unused-vars */
 const listItemAttrs = ({
   name, label, children, ...rest
 }: ListItemAttrs) => rest;
+/* eslint-enable @typescript-eslint/no-unused-vars */
 </script>
 
 <style lang="scss">
-@use "../../../styles/functions";
+@use "../../../styles/mixins";
 
 .ui-list {
-  padding: 0;
-  margin: 0;
+  $element: list;
+
+  @include mixins.use-logical($element, padding, 0);
+  @include mixins.use-logical($element, margin, 0);
+
   list-style-type: none;
 }
 </style>
```

Comment:
should be only 

```
<style lang="scss">
.ui-list {
  padding: 0;
  margin: 0;
  list-style-type: none;
}
</style>
```
we don't need CSS var here. We need only reset default styles of `ul` element